Makefile

```diff
@@ -332,5 +332,32 @@ logs:
 	socat pty,link=/tmp/ttyvnrf,waitslave tcp:127.0.0.1:8000 & disown
 	picocom /tmp/ttyvnrf -b 115200
 
-flash_with_gdb:
-	arm-none-eabi-gdb -e _build/nrf52832_xxaa_release.out -x flash.gdb
\ No newline at end of file
+daplink_erase_flash:

```

Comment:
Can you add in the README also an entry explaining this command?

Reply:
done